specification.rst

```diff
@@ -2768,8 +2768,8 @@ container-title
     recording; the session title for multi-part presentation at a conference)
 
 container-title-short
-    Short/abbreviated form of "container-title" (also accessible through the
-    "short" form of the "container-title" variable)
+    Short/abbreviated form of "container-title";
+    Deprecated use ``variable="container-title" form="short"`` instead
```

Comment:
Is capitalizing the first word after a semicolon common?

Reply:
To make the type documentation clearer, I uppercased and started a new line when there are distinct clauses/ideas being expressed. I suppose we could also force the compiled HTML to have line breaks there: https://stackoverflow.com/questions/51198270/how-do-i-create-a-new-line-with-restructuredtext